app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1695,22 +1695,25 @@ table li .account_details {
 }
 
 .back-link {
-  border: 1px solid red;
   text-align: center;
   font-size: 40px;
   margin: 10px auto;
   font-family: 'Cardinal', Arial, sans-serif;
 }
 
+. add-to-cart {
+  text-align: center;
+  padding-top: 20px;
+  background: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.